### DISCUSSION TITLE
Error al exportar canales

### DISCUSSION THREAD

**Discussion by gmartincan** [Paco8/plugin.video.movistarplus#41] [General]
Hola, estoy intentando configurar en smart TV LG pero a la hora de exportar los canales me tira un error que no sé cuál puede ser. El mensaje indica leer el error en el registro pero no sé cómo acceder al mismo para ver cuál es.
Gracias

**Comment by Paco8** [comment]
Hay un addon en el repositorio oficial que se llama Kodi logfile uploader, con el que puedes ver o subir a internet el registro.
